tests/load/pipeline/test_drop.py

```diff
@@ -523,7 +523,7 @@ def test_drop_state_only(destination_config: DestinationTestConfiguration) -> No
 
 def test_drop_first_run_and_pending_packages() -> None:
     """Attempts to drop before pipeline runs and when partial loads happen"""
-    pipeline = dlt.pipeline("drop_test_" + uniq_id(), destination="dummy")
+    pipeline = dlt.pipeline("drop_test_" + uniq_id(), destination="duckdb")
```

Comment:
I changed this because this test was previously failing with a configuration error for the dummy destination. now another test is failing with the same error, so it must be something that happens only the first time the dummy is used? does not happen locally on my mac.